saleor/product/models.py

```diff
@@ -392,6 +403,20 @@ def __str__(self):
     def quantity_available(self):
         return max(self.quantity - self.quantity_allocated, 0)
 
+    def get_cost_price(self):
+        zero_price = Price(0, 0, currency=DEFAULT_CURRENCY)
+        if not self.cost_price:
+            return zero_price
+        return self.cost_price
+
+    def get_margin_for_variant(self):
```

Comment:
I have splited get_sots_data to two methods.
Not sure if get_margin_for_variant shouldn't be called get_margin.
@patrys @mociepka @elwoodxblues what do you think?

Reply:
How about `get_gross_margin`?